client/src/pages/Search.js

```diff
@@ -62,7 +61,8 @@ export default function Search() {
       </Form>
       <div>
         {searchedAlbums.length ? searchedAlbums.map((album, i) => {
-          return <SearchAlbums key={i} album={album}/>
+          console.log(album)
+          return <SearchAlbums key={i} album={album} id={album.albumId}/>
```

Comment:
I need the albumId in order to  compare it against the album id from the me query